src/sentry/integrations/jira/integration.py

```diff
@@ -570,6 +572,7 @@ def build_dynamic_field(self, field_meta, group=None):
         return fkwargs
 
     def get_issue_type_meta(self, issue_type, meta):
+        self.parse_jira_issue_metadata(meta)
```

Comment:
Adding this here will allow us to gather some error reporting around customer configs that we may not have accounted for. I figure leaving this in for a few days should be enough.